app/src/main/java/com/nervousfish/nervousfish/activities/QRExchangeKeyActivity.java

```diff
@@ -69,6 +69,8 @@ protected void onCreate(final Bundle savedInstanceState) {
         final KeyPair pair = keyGenerator.generateRSAKeyPair("test");
         publicKey = pair.getPublicKey();
 
+        showQRCode();
+
```

Comment:
Could you remove the additional enter 😇 

Also, this PR seems like the perfect oppertunity to resolve the `//TODO:` [a few lines up](https://github.com/ericcornelissen/NervousFish/pull/224/files#diff-3ee102e75053006952e1d321d14f640dR67) 😄